README.md

```
@@ -1,247 +1,42 @@
 <div align="center">
 <a href="https://boundaryml.com?utm_source=github" target="_blank" rel="noopener noreferrer">
   <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="fern/assets/baml-lamb-white.png">
-    <img src="fern/assets/baml-lamb-white.png" height="64" id="top">
+    <source media="(prefers-color-scheme: dark)" srcset="fern/assets/baml-logo-white.png">
+    <img src="fern/assets/baml-logo-white.png" height="220" id="top">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)